transactions/views.py

```diff
@@ -47,3 +52,29 @@ class TransactionDelete(DeleteView):
 
 class TransactionList(ListView):
     model = Transaction
+
+
+class TransactionDetail(DetailView):
+    model = Transaction
+
+
+class Transfer(FormView):
+    form_class = TransferForm
+    template_name = 'transactions/transfer.html'
+    success_url = 'wallets/'
+
+    def form_valid(self, form):
```

Comment:
dadaj walidację tak aby nie mozna bedzie zrobić przelewu z wallet_1 na wallet_1, zobacz jak to działa teraz